versions/3.0.md

```diff
@@ -44,7 +44,7 @@ Some examples of possible media type definitions:
   application/vnd.github.v3.patch
 ```
 ##### <a name="httpCodes"></a>HTTP Status Codes
-The HTTP Status Codes are used to indicate the status of the executed operation. The available status codes are described by [RFC 7231](http://tools.ietf.org/html/rfc7231#section-6) and in the [IANA Status Code Registry](http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
+The HTTP Status Codes are used to indicate the status of the executed operation. The available status codes are initially defined by [RFC 7231](http://tools.ietf.org/html/rfc7231#section-6) and registered status codes are listed in the [IANA Status Code Registry](http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
```

Comment:
On 2016-07-04 19:59, Paŭlo Ebermann wrote:
>>  ##### <a name="httpCodes"></a>HTTP Status Codes
>> -The HTTP Status Codes are used to indicate the status of the executed operation. The available status codes are described by [RFC 7231](http://tools.ietf.org/html/rfc7231#section-6) and in the [IANA Status Code Registry](http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
>> +The HTTP Status Codes are used to indicate the status of the executed operation. The available status codes are initially defined by [RFC 7231](http://tools.ietf.org/html/rfc7231#section-6) and registered status codes are listed in the [IANA Status Code Registry](http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
>
> I'm not sure how this is better than the original.
> Could you add some sentences of justification?

trying to make it clearer that the available status codes are not 
defined by RFC 7231, but instead by the registry which has been 
established (and initialized with a first set of entries) by that RFC. 
if you want to know the available status codes, you have to look at the 
registry, not the RFC.